src/app/pages/MediaArticlePage/SecondaryColumn.tsx

```diff
@@ -2,12 +2,20 @@
 /** @jsx jsx */
 import path from 'ramda/src/path';
 import { jsx } from '@emotion/react';
-import { articleDataPropTypes } from '#models/propTypes/article';
+import { articleDataPropTypes } from '#app/models/propTypes/article';
 import LatestMediaSection from './PagePromoSections/LatestMediaSection';
 import styles from './MediaArticlePage.styles';
+import { LatestMedia } from './PagePromoSections/LatestMediaSection/types';
 
-const SecondaryColumn = ({ pageData }) => {
-  const latestMediaContent = path(['secondaryColumn', 'latestMedia'], pageData);
+type MediaArticlePageProps = {
+  pageData: typeof articleDataPropTypes;
```

Comment:
Ah.. I was wondering if we could put the older prop types like `articleDataPropTypes` in the 🗑️ - is this something we might do eventually (but not as part of this work)?

Reply:
Yea its a bit of tricky one as these objects as _massive_ in reality, so these are the best types we have. I've had brief conversations with FABL to see if we can decouple the FABL types library from FABL to make it more generally available, as that would give us a close end-to-end type safety.

For now though you're right, I've remove these and just use a generic `object` type as we're not really peeking into the `pageData` too much here anyway.